.azuredevops/backstage.yaml

```diff
@@ -271,7 +271,7 @@ properties:
             secretRef: cluster6-env
 
           - name: ADP_PORTAL_TEMPLATE_VERSION
-            value: '1.0.3'
+            value: '1.0.4'
```

Comment:
new template will prob be 1.1.0 ? minor update for a new template

Reply:
@DaRoszko done. Updated it to 1.1.0